x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -307,6 +307,8 @@ export interface Tree {
    * All events from children, ancestry, origin, and the alert in a single array
    */
   allEvents: Event[];
+  startTime: Date;
```

Comment:
Added these so the integration tests could easily identify the `to` and `from` time range fields.